components/auth/Join.tsx

```diff
@@ -92,6 +95,21 @@ const Join = () => {
           error={formik.touched.password ? formik.errors.password : undefined}
           onChange={formik.handleChange}
         />
+
+
+        {/* ADDING CHECKBOX HERE */}
```

Comment:
You can use `Checkbox` component directly here, no need to create a new component for it. We'll create a new component for `Checkbox` when we need to add some extra functionality to it.